eventmesh-rest/src/main/java/org/apache/eventmesh/rest/OpenSchema/SchemaService/OpenSchemaService.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.eventmesh.rest.OpenSchema.SchemaService;
+
+import org.apache.eventmesh.rest.OpenSchema.Bean.Compatibility;
+import org.apache.eventmesh.rest.OpenSchema.Bean.Schema;
+import org.apache.eventmesh.rest.OpenSchema.Bean.Subject;
+import org.apache.eventmesh.spi.EventMeshSPI;
+
+import java.util.List;
+
+@EventMeshSPI(isSingleton = true)
+public interface OpenSchemaService {
+
+    // Subject
```

Comment:
It is better to use stander java annotation in spi interface.
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html.

Reply:
Hi, thanks for your review, what standard annotation do you think I should use?  I set this annotation the same as annotation of  ```org.apache.eventmesh.api.consumer.MeshMQPushConsumer``` because they are all plugins in runtime.

Reply:
Yes, the annotation is right.
It's my wrong, I just mean you can add the standard comment annotation to write comment in interface, then others can  easily know the method of the interface, and add implementation easily.